providers/nitro/nitro-helper/Cargo.toml

```
@@ -35,3 +35,5 @@ tracing = "0.1"
 tracing-subscriber = "0.3"
 tracing-core = "0.1"
 vsock = "0.2"
+aws-config = "0.9.0"
+aws-types = "0.9.0"
```

```suggestion
aws-config = "0.9"
aws-types = "0.9"
```

plus perhaps `cargo sort` to put it at the front?